src/htm/algorithms/SpatialPooler.cpp

```diff
@@ -473,14 +477,40 @@ const vector<SynapseIdx> SpatialPooler::compute(const SDR &input, const bool lea
     bumpUpWeakColumns_();
     updateBoostFactors_();
     if (isUpdateRound_()) {
+      const UInt old_inhibitionRadius_ = inhibitionRadius_;
       updateInhibitionRadius_();
+      if (inhibitionRadius_ != old_inhibitionRadius_) { 
+	      calculateWrapAroundNeighbors();
```

Comment:
nit, make this `UInt calculateWrapAroundNeighbors() const` `

Reply:
I tried, but got a few errors like this:

> SpatialPooler.cpp:498:30: error: assignment of member 'htm::SpatialPooler::wrapAroundNeighbors_' is a read-only object
> wrapAroundNeighbors_ =1/

Am I doing something wrong?

Reply:
Oh, I see what you mean. I was trying to follow what I thought was some sort of standard in the code and just set the wrapAroundNeighbors_ directly inside the function, as is done with updateInhibitionRadius_(). But I'll change it, no problem.

Reply:
Implemented in #783